_store/hr/infeeeee-kimai2-cmd.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Command line client
-intro: "Cross platform command line client for Kimai, written with Node.js"
+title: Klijent naredbenog retka
```

Comment:
An IT guy would understand "Command line client" and I doubt that anyone else except an IT professional would look for a console tool 😁 
So I wonder if it is helping to translate it in this case or if the translation makes it harder to find. In German I would not translate it. I trust your choice(!), just wondering ...